### DISCUSSION TITLE
How to calculate percentage of observations

### DISCUSSION THREAD

**Discussion by mvarewyck** [inbo/reporting-rshiny-grofwildjacht#249] [Ideas]
We should carefully define how to calculate "% gekend aantal"
Previously we excluded observations with missing 'aantal_embryos_bron' such that we obtained a percentage of 100%. If we take also into account these observations, which seems more correct to my opinion, we now obtain only 61%.

![currentApp](https://user-images.githubusercontent.com/22143743/139093345-062ca67f-1a7c-441e-8b26-b2e8e3079db6.png)

![newApp](https://user-images.githubusercontent.com/22143743/139093354-4e9bec61-2be3-4a60-8d1e-80d95d2927bd.png)

Similarly, e.g. for "FIGUUR: Leeggewicht per leeftijdscategorie en geslacht (INBO of Meldingsformulier)" if we deselect some types the percentage goes down, as the total number doesn't go down.

![withoutJongvolwassen](https://user-images.githubusercontent.com/22143743/139095001-635b346d-2ae5-4a53-9f23-474c8ed5af41.png)

![withJongvolwassen](https://user-images.githubusercontent.com/22143743/139095020-4c7c6a2c-a42f-43fd-9eef-a287de63d10b.png)

**Comment by SanderDevisscher** [comment]
@mvarewyck you make valid arguments.
In case of the embryo's I would take the total number of animals with known number of embryos divided by the total number of females.  @jimcasaer do you think it is best to cap the total number of females by age ea only females of a certain age and older? or should we take all females? 

In case of the "FIGUUR: Leeggewicht per leeftijdscategorie en geslacht (INBO of Meldingsformulier)" the percentage should be the fraction of the total number of animals with age and sex information determined by the respective sources divided by the total number of animals shot, independant of the types selected. 

I'm converting this into a issue for the februari update

**Reply by SanderDevisscher** [reply]
#251
